Modules/Challenges/test/Challenges.API.Tests.Integration/Features/CreateChallenge.feature

```diff
@@ -0,0 +1,37 @@
+﻿@Integration
+Feature: Create a Challenge
+
+User creates a Challenge
+
+Scenario: Creating a Challenge as an anonymous user
+	Given the user is unauthenticated
+	When a POST request is sent to the Challenges endpoint
```

Comment:
Shouldn't it be called `When he sends a POST request to the Challenges endpoint`?

Reply:
I followed the examples they have in this specflow blog post:
https://specflow.org/challenges/test-apis-with-gherkin/